components/org.wso2.carbon.siddhi.error.handler.core/src/main/java/org/wso2/carbon/siddhi/error/handler/core/execution/RePlayer.java

```diff
@@ -24,9 +24,9 @@
 import io.siddhi.core.stream.input.InputHandler;
 import io.siddhi.core.stream.input.source.Source;
 import io.siddhi.core.util.error.handler.model.ErrorEntry;
+import io.siddhi.core.util.error.handler.model.ReplayableTableRecord;
 import io.siddhi.core.util.error.handler.store.ErrorStore;
-import io.siddhi.core.util.error.handler.util.ErrorOccurrence;
-import io.siddhi.core.util.error.handler.util.ErrorHandlerUtils;
+import io.siddhi.core.util.error.handler.util.*;
```

Comment:
Let's use single class imports instead of `*`

Reply:
Fixed in 379f207